From ab01fa5a2fb551e4dc8e5a4de84792f485692748 Mon Sep 17 00:00:00 2001
From: Sugar Zhang <sugar.zhang@rock-chips.com>
Date: Mon, 14 May 2018 11:51:41 +0800
Subject: [PATCH] arm64: rockchip_defconfig: enable
 CONFIG_SND_SOC_ROCKCHIP_FORCE_SRAM

This patch depends on iram node and 'rockchip,force-iram' property.

the example on rk3399 to active this feature is as follows:

iram: sram@ff8d0000 {
	compatible = "mmio-sram";
	reg = <0x0 0xff8d0000 0x0 0x20000>; /* 128k */
};

&dmac_bus {
	iram = <&iram>;
	rockchip,force-iram;
};

if the 'rockchip,force-iram' not defined, fallthrough to normal usage.

Change-Id: I625a50835a6ea91dd2a66698bec087551be8a12c
Signed-off-by: Sugar Zhang <sugar.zhang@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 981d6e3651f3..35c01ebb9ae0 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -538,6 +538,7 @@ CONFIG_SND_VERBOSE_PRINTK=y
 CONFIG_SND_USB_AUDIO=y
 CONFIG_SND_SOC=y
 CONFIG_SND_SOC_ROCKCHIP=y
+CONFIG_SND_SOC_ROCKCHIP_FORCE_SRAM=y
 CONFIG_SND_SOC_ROCKCHIP_I2S_TDM=y
 CONFIG_SND_SOC_ROCKCHIP_PDM=y
 CONFIG_SND_SOC_ROCKCHIP_SPDIF=y
-- 
2.35.3

